### ISSUE TITLE
First and Last Name vs Full name?

### ISSUE THREAD

**Issue by amercader** [keitaroinc/ckanext-saml2auth#11]
I noticed that there is a requirement to provide separate fields for First and Last Name on the SAML fields.

Is there a reason for this? It is not always the case that you will get these as part of a SAML payload, and a single field for the full name is more common in my experience. Besides CKAN itself only supports `fullname` on the User model, and the code seems to do `'fullname': '{0} {1}'.format(firstname, lastname)` anyway.

Perhaps there could be support for a `ckanext.saml2auth.user_fullname` option that can be used instead of `ckanext.saml2auth.user_firstname` + `ckanext.saml2auth.user_lastname`?

Happy to submit a PR if that sounds good

**Comment by mbocevski** [comment]
When we were designing this extension we were looking into the most commonly used providers that users would use for SAML, such as Google Workspaces, Azure AD, Okta, Onelogin etc. and we noticed that in some cases fullname is not present as a built-in attribute in some services, therefore we decided to split the mapping on first name, last name cause all the services that we looked into had built-in support for those attributes. That was the reasoning behind this. 

I don't think it's a big deal of having two attribute mappings that are joined, but I'm happy if we can make it optional? Either map fullname or first name+last name?
